### DISCUSSION TITLE
GitLab Sync

### DISCUSSION THREAD

**Discussion by mii-shell** [six7/figma-tokens#374] [General]
Is it possible to sync Figma Tokens with a GitLab repo as well? If not, is this feature planned some time in the future?

**Comment by six7** [comment]
You can read tokens with the URL sync and the gitlab rest api.

There might be a sync in the future if teams request it, but no immediate plans.

**Reply by shanilp7** [reply]
Hey, just curious if this has been elevated or requested more? 
I'm looking to use Figma Tokens but our devs use Gitlab not Github and it would be awesome to be able sync between the two.

**Reply by six7** [reply]
Just released! 🙏

**Comment by six7** [comment]
Just released Version 97 with GitLab sync 🎉 Find it in the Settings tab!
